src/plugin.ts

```diff
@@ -1,8 +1,10 @@
+// @ts-ignore
```

Comment:
Can you avoid `// @ts-ignore` by adding `@types/html-escaper` to devDependeincies using v3 of `html-escaper`?

https://www.npmjs.com/package/@types/html-escaper

If you cannot do it for some reason, you can leave it as it is.

Reply:
yeah, i used this because something else in the project i was working with depended on v2, so i figured i should play it safe. I added some stuff in the test to verify that html-escaper didn't have regressions in v3.

looking at the changelog & [diff](https://github.com/WebReflection/html-escaper/compare/v2...master), v3 just deprecates older platforms, which should be fine for a typedoc plugin (it doesn't need to run in old browsers)